cocos/game/game.ts

```diff
@@ -25,7 +25,7 @@
 
 import { DEBUG, EDITOR, NATIVE, PREVIEW, TEST, EDITOR_NOT_IN_PREVIEW } from 'internal:constants';
 import { systemInfo } from 'pal/system-info';
-import { findCanvas, loadJsFile } from 'pal/env';
+import { findCanvas, loadJsFile } from '@pal/env';
```

Comment:
I thought pal is a module, why can import from its subclass? Or `pal/env` is a module, and pal contains more modules?

Reply:
all the pal modules can be merged together, but it need times to organize module structure

`@pal/env` `@pal/utils` is the smallest module I could thought.

Reply:
Ok.